assets/javascripts/discourse/components/rag-options.gjs

```diff
@@ -22,6 +23,22 @@ export default class RagOptions extends Component {
       : i18n("discourse_ai.rag.options.show_indexing_options");
   }
 
+  get visionLlms() {
+    return this.args.llms.filter((llm) => llm.vision_enabled);
+  }
+
+  get visionLlmId() {
+    return this.args.model.rag_llm_model_id || "blank";
+  }
```

Comment:
```suggestion
  get visionLlmId() {
    return this.args.model.rag_llm_model_id ?? "blank";
  }
```

I don't know if you expect 0 or negative ids, but I recommend this pattern for this kind of defaults, it's safer.

![Screenshot 2025-02-12 at 11 57 29](https://github.com/user-attachments/assets/ba85b89a-7e4c-492e-9e31-9a75cd440569)

Reply:
nice catch!

Reply:
also TIL